### PR TITLE
Created pi.hi.md

### PR DESCRIPTION
**Pull request by AdityaGarg00**
Translated pi.md file into local hindi

### CODE REVIEW

**Review by Anurag-0-1-A — CHANGES_REQUESTED** [review]

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+अपने पाई पर ग्रोव बेस हैट स्थापित करें और पाई को कॉन्फ़िगर करें
+
+1. ग्रोव बेस हैट को अपने पाई से कनेक्ट करें। हैट का सॉकेट पाई पर सभी जी.पी.आई.ओ पिनों पर फिट बैठता है, पाई पर मजबूती से बैठाने के लिए सभी तरह से पिन को नीचे की ओर खिसकायें। यह पाई के ऊपर बैठता है उसे ढकते हुए।
+
+    ![ग्रोव हैट फिट करना](../../../../images/pi-grove-hat-fitting.gif)
+
+1. तय करें कि आप अपने पाई को कैसे प्रोग्राम करना चाहते हैं, और नीचे दिए गए संबंधित अनुभाग पर जाएं:
+
+    * [सीधे अपने पाई पर काम करें](#सीधे-अपने-पाई-पर-काम-करें)
```
[the remark this answers is not in the document]
Comment (AdityaGarg00) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:25]: These are having the links to below sections of the file

File: 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+एक बार पीआई कॉन्फ़िगर हो जाने के बाद, आप अपने कंप्यूटर से विजुअल स्टूडियो कोड (वीएस कोड) का उपयोग करके इससे जुड़ सकते हैं - यह एक मुफ्त डेवलपर टेक्स्ट एडिटर है जिसका उपयोग आप अपने डिवाइस कोड को पायथन में लिखने के लिए करेंगे।
+
+##### कार्य - रिमोट एक्सेस के लिए वीएस कोड कॉन्फ़िगर करें
+
+आवश्यक सॉफ़्टवेयर स्थापित करें और दूरस्थ रूप से अपने पाई से कनेक्ट करें।
+
+1. [वीएस कोड प्रलेखन](https://code.visualstudio.com?WT.mc_id=academic-17441-jabenn) का पालन करके अपने कंप्यूटर पर वीएस कोड स्थापित करें।
+
+1. आवश्यक घटकों को स्थापित करने के लिए [एसएसएच दस्तावेज़ीकरण का उपयोग कर वीएस कोड रिमोट डेवलपमेंट](https://code.visualstudio.com/docs/remote/ssh?WT.mc_id=academic-17441-jabenn) में दिए गए निर्देशों का पालन करें।
```
[the remark this answers is not in the document]
Comment (AdityaGarg00) [reply 1-getting-started/lessons/1-introduction-to-iot/translations/pi.hi.md:188]: Dastavejikaran is simpler word

**Review by AdityaGarg00 — COMMENTED** [review]
I have done the required changes

**Comment by jimbobbennett** [comment]
@AdityaGarg00 - I can't see the merge options, I think this is another issue from me making the repo public. Can you re-fork, copy your changes over and re-submit a new PR, then close this one please.

**Pull request closed without merging**